actioncable.gemspec

```diff
@@ -18,8 +18,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'actionpack',       '>= 4.2.0'
   s.add_dependency 'faye-websocket',   '~> 0.10.0'
   s.add_dependency 'websocket-driver', '~> 0.6.1'
-  # Use 0.16.0 until https://github.com/celluloid/celluloid/issues/637 is resolved
-  s.add_dependency 'celluloid',        '~> 0.16.0'
```

Comment:
Because we can't say `0.16.0` OR `> 0.17.2`

We could say `> 0.16.0` and then add a note to the readme that `0.17.0` and `0.17.1` have breaking issues with Spring.

The other downside to supporting both `0.16` and `0.17` is that the primary require for celluloid has moved from `require 'celluloid'` to `require 'celluloid/current'` which would mean we'd either need to force celluloid to run in what they term "fallback" mode and put up with the warnings, or put version switching code in the many places celluloid is required.